src/packages/database/database_manager.py

```diff
@@ -52,6 +52,53 @@ class property, and the DatabaseManage.query() method used to perform a query.
         >>> dbm.query(query, ('tuple','of','values'))# doctest: +SKIP
 
     """
+    @classmethod
+    @CONFIG_MARKER
+    def rehash_handler(cls, data: Dict):
+        """
+        Apply new configuration data
+
+        Args:
+            data (typing.Dict): new configuration data to apply.
+
+        """
+        cls._config = data
+
+    _config: ClassVar[Dict] = {}
```

Comment:
[Style] [Non-Blocking] Wouldn't this be better placed at the top of the class?

Reply:
possibly, though it makes no difference functionally. 
Il see to moving it with other pending changes.